### ISSUE TITLE
reactquestiondropdown - oninput issue on mobile (firefox)

### ISSUE THREAD

**Issue by gperreymond** [surveyjs/surveyjs#392]
Hi the team,

I understand why you put oninput in a select but in fact on mobile there is a problem.
On chrome it's ok because on mobile, chrome associate oninput and onchange, but for firefox it's a problem because he dissociates the two.

On firefox it's like this :

```html
<select id="theid" onchange=null oninput="bound()">
```

So the event oninput will be never fired, and onchange the same :)

Your code
```html
<select id={this.question.inputId} className={this.css.control} value={this.state.value} onChange={onChange} onInput={this.handleOnChange}>

```

### Specify your

* browser: firefox
* browser version: 53
* surveyjs platform (angular or react or jquery or knockout or vue): react
* surveyjs version: all

**Comment by dmitrykurmanov** [comment]
fix will be in release next to 0.12.30

**Comment by gperreymond** [comment]
Exellent, thanks a lot.
